testing/scenario_app/ios/Scenarios/Scenarios.xcodeproj/project.pbxproj

```diff
@@ -1046,7 +1046,7 @@
 				INFOPLIST_FILE = ScenariosShare/Info.plist;
 				INFOPLIST_KEY_CFBundleDisplayName = ScenariosShare;
 				INFOPLIST_KEY_NSHumanReadableCopyright = "Copyright © 2023 flutter. All rights reserved.";
-				IPHONEOS_DEPLOYMENT_TARGET = 16.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 17.0;
```

Comment:
I wonder if https://github.com/flutter/engine/pull/46329 added this override on purpose?  If you remove it (highlight **iOS Deployment Target** and cmd-delete in Xcode) does it pass with the inherited iOS 12?
<img width="846" alt="Screenshot 2024-01-17 at 3 20 40 PM" src="https://github.com/flutter/engine/assets/682784/e0689fcd-0034-4e6c-82c4-eae2b6d1dd81">

Reply:
Everything still passes for me if I remove it. I'll go ahead a remove it in this PR